src/ert/_c_wrappers/enkf/observations/obs_vector.py

```diff
@@ -142,7 +139,7 @@ def getConfigNode(self) -> EnkfConfigNode:
         return self._get_config_node().setParent(self)
 
     def hasData(self, active_mask, fs) -> bool:
-        return self._has_data(active_mask, fs)
+        raise NotImplementedError("Not yet implemented")
```

Comment:
Suggest an issue for this, and solving afterwards

Reply:
Actually, can only find this used here: https://github.com/equinor/semeio/pull/494, so actually suggest just removing it